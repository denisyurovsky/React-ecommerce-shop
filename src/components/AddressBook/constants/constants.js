export const LISTBOX_PADDING = 8;
export const DOUBLE_LISTBOX_PADDING = LISTBOX_PADDING * 2;
export const LIMIT_NUMBER_ITEMS = 8;
export const ITEM_SIZE_LG = 48;
export const ITEM_SIZE_MD = 36;
