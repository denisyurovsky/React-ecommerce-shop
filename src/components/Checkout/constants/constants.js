export const PANEL = {
  DELIVERY_ADDRESS: 'Delivery Address',
  PERSONAL_INFORMATION: 'Personal Information',
  PERSONAL_ADDRESS: 'Personal Address',
  PAYMENT_METHOD: 'Payment method',
  ORDER_CONFIRMATION: 'Order confirmation',
};
