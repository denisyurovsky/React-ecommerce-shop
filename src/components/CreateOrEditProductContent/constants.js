export const PRICE_ERROR = 'Only numbers';
export const DESCRIPTION_ERROR = 'Only latin characters';
export const LENGTH_ERROR = 'Should contain at least one symbol';
export const DISCOUNT_ERROR = 'Discount should be less than price';
export const SELECTION_ERROR = 'Choose category';
