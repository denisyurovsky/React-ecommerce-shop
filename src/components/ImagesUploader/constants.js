export const LOSS_OF_IMAGES = 'We lost images';

export const STATUS = {
  CROPPED: 'cropped',
  CROPPING: 'cropping',
  IDLE: 'idle',
};

export const TEST_IMG_URL = 'http://someURL';
