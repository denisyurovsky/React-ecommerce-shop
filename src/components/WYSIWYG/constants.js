export const TYPES = {
  LINK: 'LINK',
  CELL: 'CELL',
  EMAIL: 'EMAIL',
  TABLE: 'TABLE',
};

export const table = {
  MAX_ROWS: 20,
  COLUMNS: 2,
  HEADER: 'Product details',
  INPUT_ERROR: 'Max 20 rows',
};

export const MAX_LENGTH_MESSAGE = 'Max size is reached';
export const LINK_PREDEFINED_TEXT = 'https://';
export const EDITOR_MAX_SYMBOLS = 1000;
export const EDITOR_WARN_SYMBOLS = 50;

export const BREAKPOINTS = {
  FOOTER: 550,
  NAVIGATION: 350,
};
