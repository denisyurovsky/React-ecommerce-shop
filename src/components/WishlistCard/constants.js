export const DEFAULT_LABEL_INPUT = 'Wishlist Name';
export const ERROR_LABEL_INPUT = 'Wishlist name should be unique';
