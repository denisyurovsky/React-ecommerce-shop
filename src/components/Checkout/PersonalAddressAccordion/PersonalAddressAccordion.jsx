import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
import EditIcon from '@mui/icons-material/Edit';
import LocalShippingIcon from '@mui/icons-material/LocalShipping';
import {
  Accordion,
  AccordionDetails,
  Box,
  Typography,
  Button,
} from '@mui/material';
import classNames from 'classnames';
import { createBrowserHistory } from 'history';
import PropTypes from 'prop-types';
import React, { useContext } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { toast } from 'react-toastify';

import productsApi from '../../../api/products';
import { notificationError } from '../../../constants/constants';
import { pathNames } from '../../../constants/pathNames';
import getSelectedProductIds from '../../../helpers/getSelectedProductIds';
import { CheckoutContext } from '../../../pages/CheckoutPage/CheckoutPage';
import {
  deleteSelectedProducts,
  selectCart,
} from '../../../store/cart/cartSlice';
import { addOrder, editAddressOrder } from '../../../store/orders/ordersSlice';
import DeliveryAddress from '../../AddressBook/DeliveryAddress/DeliveryAddress';
import { PANEL } from '../constants/constants';
import AccordionSummary from '../helpers/AccordionSummary';

import styles from './PersonalAddressAccordion.module.scss';

const PersonalAddressAccordion = ({
  expanded,
  handleChangeAccordion,
  address,
  setAddress,
  handleChange,
  isPersonalAddressValid,
  isPersonalInformationValid,
  setExpanded,
  setCreatedOrderId,
  orderId,
}) => {
  const {
    country: { name: countryName },
    city,
    street,
    building,
    zip,
  } = address;
  const isPersonalAddressExpended = expanded === PANEL.PERSONAL_ADDRESS;
  const cart = useSelector(selectCart);
  const dispatch = useDispatch();
  const [disabledAccordion, setDisabledAccordion] = useContext(CheckoutContext);

  const handlePersonalAddressButton = async () => {
    setExpanded(PANEL.PAYMENT_METHOD);
    setDisabledAccordion({ ...disabledAccordion, payment: false });

    if (orderId) {
      try {
        dispatch(
          editAddressOrder({ orderId, addressData: address, addressId: null })
        );
      } catch (err) {
        toast.error(notificationError);
      }
    } else {
      try {
        const checkedProductIds = getSelectedProductIds(cart.sellers);
        const productsResponse = await productsApi.getProductsByIds(
          checkedProductIds
        );
        const res = await dispatch(
          addOrder({
            productsData: productsResponse.data,
            addressData: address,
            addressId: null,
          })
        );

        setCreatedOrderId(res.payload.id);
        createBrowserHistory().push(`${pathNames.CHECKOUT}/${res.payload.id}`);
        dispatch(deleteSelectedProducts());
      } catch (err) {
        toast.error(notificationError);
      }
    }
  };

  return (
    <Accordion
      disabled={disabledAccordion.address}
      className={classNames({
        [styles.accordionSummary]: !isPersonalAddressExpended,
        [styles.accordion]: true,
      })}
      expanded={isPersonalAddressExpended}
      onChange={handleChangeAccordion(PANEL.PERSONAL_ADDRESS)}
    >
      <AccordionSummary
        expandIcon={
          isPersonalAddressValid &&
          isPersonalInformationValid &&
          !isPersonalAddressExpended ? (
            <EditIcon />
          ) : (
            <ArrowDropDownIcon />
          )
        }
        data-testid="personalAddressSummery"
      >
        <Box className={styles.header}>
          <LocalShippingIcon className={styles.icon} />
          <Box>
            <Typography className={styles.subtitle} variant="subtitle2">
              Delivery address
            </Typography>
            {!isPersonalAddressExpended && isPersonalAddressValid && (
              <Box className={styles.body}>
                <Typography variant="caption">
                  {countryName}, {city}
                </Typography>
                <Typography variant="caption" data-testid="address">
                  {street}, {building}
                </Typography>
                <Typography variant="caption">{zip}</Typography>
              </Box>
            )}
          </Box>
        </Box>
      </AccordionSummary>
      <AccordionDetails>
        <DeliveryAddress
          handleChange={handleChange}
          address={address}
          setAddress={setAddress}
        />
        <Button
          fullWidth
          onClick={handlePersonalAddressButton}
          className={styles.button}
          variant="contained"
          disabled={!isPersonalAddressValid || !isPersonalInformationValid}
          data-testid="personalAddressButton"
        >
          Proceed
        </Button>
      </AccordionDetails>
    </Accordion>
  );
};

PersonalAddressAccordion.propTypes = {
  handleChangeAccordion: PropTypes.func.isRequired,
  address: PropTypes.object.isRequired,
  setAddress: PropTypes.func.isRequired,
  setCreatedOrderId: PropTypes.func.isRequired,
  handleChange: PropTypes.func.isRequired,
  setExpanded: PropTypes.func.isRequired,
  expanded: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
  orderId: PropTypes.number,
  isPersonalInformationValid: PropTypes.bool,
  isPersonalAddressValid: PropTypes.bool,
};

export default PersonalAddressAccordion;
