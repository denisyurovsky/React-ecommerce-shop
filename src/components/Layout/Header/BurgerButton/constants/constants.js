import { pathNames } from '../../../../../constants/pathNames';

export const LINKS = [
  {
    name: 'HOME',
    link: '/',
  },
  {
    name: 'PRODUCTS',
    link: pathNames.PRODUCTS,
  },
];
