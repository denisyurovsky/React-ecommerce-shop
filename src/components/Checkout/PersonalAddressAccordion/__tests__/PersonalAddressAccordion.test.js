import React from 'react';

import { EMPTY_ADDRESS } from '../../../../pages/AddressBookPage/constants/constants';
import renderWith from '../../../../test-utils/renderWith';
import { PANEL } from '../../constants/constants';
import PersonalAddressAccordion from '../PersonalAddressAccordion';

describe('PersonalAddressAccordion component', () => {
  it('should render valid snapshot', () => {
    const { asFragment } = renderWith(
      <PersonalAddressAccordion
        address={EMPTY_ADDRESS}
        isDeliveryAddressDisabled={false}
        isPersonalAddressValid={false}
        isPersonalInformationValid={true}
        orderId={null}
        expanded={PANEL.PERSONAL_ADDRESS}
        handleChangeAccordion={jest.fn()}
        setAddress={jest.fn()}
        handleChange={jest.fn()}
        setIsPaymentMethodDisabled={jest.fn()}
        setExpanded={jest.fn()}
        setCreatedOrderId={jest.fn()}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
