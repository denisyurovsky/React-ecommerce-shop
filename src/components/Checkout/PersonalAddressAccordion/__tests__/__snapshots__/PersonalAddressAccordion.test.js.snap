// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalAddressAccordin component should render valid snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded Mui-expanded MuiAccordion-gutters accordion css-1elwnq4-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-expanded="true"
      class="MuiButtonBase-root MuiAccordionSummary-root Mui-expanded MuiAccordionSummary-gutters css-24vt7b-MuiButtonBase-root-MuiAccordionSummary-root"
      data-testid="personalAddressSummery"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content Mui-expanded MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <div
          class="header MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium icon css-i4bv87-MuiSvgIcon-root"
            data-testid="LocalShippingIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M20 8h-3V4H3c-1.1 0-2 .9-2 2v11h2c0 1.66 1.34 3 3 3s3-1.34 3-3h6c0 1.66 1.34 3 3 3s3-1.34 3-3h2v-5l-3-4zM6 18.5c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5zm13.5-9 1.96 2.5H17V9.5h2.5zm-1.5 9c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5z"
            />
          </svg>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 subtitle css-ati914-MuiTypography-root"
            >
              Delivery address
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper Mui-expanded css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ArrowDropDownIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="m7 10 5 5 5-5z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-entered css-pwcg7p-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            class="MuiAccordion-region"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <div
                class="wrapper MuiBox-root css-0"
              >
                <div
                  aria-expanded="false"
                  class="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon country css-16awh2u-MuiAutocomplete-root"
                  data-testid="autocomplete-country"
                  role="combobox"
                >
                  <div
                    class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="false"
                      for="mui-2"
                      id="mui-2-label"
                    >
                      Country
                    </label>
                    <div
                      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-ghsjzk-MuiInputBase-root-MuiInput-root"
                    >
                      <input
                        aria-autocomplete="list"
                        aria-describedby="mui-2-helper-text"
                        aria-invalid="false"
                        autocapitalize="none"
                        autocomplete="off"
                        class="MuiInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-1x51dt5-MuiInputBase-input-MuiInput-input"
                        id="mui-2"
                        spellcheck="false"
                        type="text"
                        value=""
                      />
                      <span
                        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorInherit css-62e83j-MuiCircularProgress-root"
                        role="progressbar"
                        style="width: 20px; height: 20px;"
                      >
                        <svg
                          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
                          viewBox="22 22 44 44"
                        >
                          <circle
                            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
                            cx="44"
                            cy="44"
                            fill="none"
                            r="20.2"
                            stroke-width="3.6"
                          />
                        </svg>
                      </span>
                      <div
                        class="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
                      >
                        <button
                          aria-label="Open"
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-qzbt6i-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
                          tabindex="-1"
                          title="Open"
                          type="button"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="ArrowDropDownIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M7 10l5 5 5-5z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </button>
                      </div>
                    </div>
                    <p
                      class="MuiFormHelperText-root MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
                      id="mui-2-helper-text"
                    >
                      <span
                        class="notranslate"
                      >
                        â€‹
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  aria-expanded="false"
                  class="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon css-16awh2u-MuiAutocomplete-root"
                  data-testid="autocomplete-city"
                  role="combobox"
                >
                  <div
                    class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary Mui-disabled css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="false"
                      for="mui-4"
                      id="mui-4-label"
                    >
                      City
                    </label>
                    <div
                      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary Mui-disabled MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-ghsjzk-MuiInputBase-root-MuiInput-root"
                    >
                      <input
                        aria-autocomplete="list"
                        aria-describedby="mui-4-helper-text"
                        aria-invalid="false"
                        autocapitalize="none"
                        autocomplete="off"
                        class="MuiInput-input MuiInputBase-input Mui-disabled MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-1x51dt5-MuiInputBase-input-MuiInput-input"
                        disabled=""
                        id="mui-4"
                        spellcheck="false"
                        type="text"
                        value=""
                      />
                      <div
                        class="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
                      >
                        <button
                          aria-label="Open"
                          class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-qzbt6i-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
                          disabled=""
                          tabindex="-1"
                          title="Open"
                          type="button"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="ArrowDropDownIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M7 10l5 5 5-5z"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                    <p
                      class="MuiFormHelperText-root Mui-disabled MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
                      id="mui-4-helper-text"
                    >
                      <span
                        class="notranslate"
                      >
                        â€‹
                      </span>
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="false"
                  for="street"
                  id="street-label"
                >
                  Street
                </label>
                <div
                  class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-ghsjzk-MuiInputBase-root-MuiInput-root"
                >
                  <input
                    aria-describedby="street-helper-text"
                    aria-invalid="false"
                    class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                    id="street"
                    type="text"
                    value=""
                  />
                </div>
                <p
                  class="MuiFormHelperText-root MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
                  id="street-helper-text"
                >
                  <span
                    class="notranslate"
                  >
                    â€‹
                  </span>
                </p>
              </div>
              <div
                class="wrapper MuiBox-root css-0"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root building css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for="building"
                    id="building-label"
                  >
                    Building
                  </label>
                  <div
                    class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-1480iag-MuiInputBase-root-MuiInput-root"
                  >
                    <input
                      aria-describedby="building-helper-text"
                      aria-invalid="false"
                      class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                      id="building"
                      type="text"
                      value=""
                    />
                  </div>
                  <p
                    class="MuiFormHelperText-root MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
                    id="building-helper-text"
                  >
                    <span
                      class="notranslate"
                    >
                      â€‹
                    </span>
                  </p>
                </div>
                <div
                  class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for="flat"
                    id="flat-label"
                  >
                    Flat
                  </label>
                  <div
                    class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-1480iag-MuiInputBase-root-MuiInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                      id="flat"
                      type="text"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="false"
                  for="mui-5"
                  id="mui-5-label"
                >
                  Zip
                </label>
                <div
                  class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-ghsjzk-MuiInputBase-root-MuiInput-root"
                >
                  <input
                    aria-describedby="mui-5-helper-text"
                    aria-invalid="false"
                    class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                    id="mui-5"
                    type="text"
                    value=""
                  />
                </div>
                <p
                  class="MuiFormHelperText-root MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
                  id="mui-5-helper-text"
                >
                  <span
                    class="notranslate"
                  >
                    â€‹
                  </span>
                </p>
              </div>
              <button
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButtonBase-root Mui-disabled button css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                data-testid="personalAddressButton"
                disabled=""
                tabindex="-1"
                type="button"
              >
                Proceed
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="Toastify"
  />
</DocumentFragment>
`;
